js/client.js

```diff
@@ -2199,6 +2236,73 @@
 		}
 	});
 
+	Config.groups = Config.groups || {
+		'~': {
+			name: "Administrator (~)",
+			type: 2,
+			order: 1,
+		},
+		'#': {
+			name: "Room Owner (#)",
+			type: 2,
+			order: 2,
+		},
+		'&': {
+			name: "Leader (&amp;)",
+			type: 2,
+			order: 3,
+		},
+		'@': {
+			name: "Moderator (@)",
+			type: 1,
+			order: 4,
+		},
+		'%': {
+			name: "Driver (%)",
+			type: 1,
+			order: 5,
+		},
+		'*': {
+			name: "Bot (*)",
+			type: 0,
+			order: 6,
+		},
+		'\u2606': {
+			name: "Player (\u2606)",
+			type: 0,
+			order: 7,
+		},
+		'\u2605': {
+			name: "Player (\u2605)",
+			type: 0,
+			order: 8,
+		},
+		'+': {
+			name: "Voice (+)",
+			type: 0,
+			order: 9,
+		},
+		' ': {
+			type: 0,
+			order: 10,
+		},
+		'!': {
+			name: "<span style='color:#777777'>Muted (!)</span>",
+			type: 0,
+			order: 11,
```

Comment:
*fat* Zarel said:
```
It's looking pretty good. I think it might be nice to keep all the default groups at 10000+, so they go below custom groups?
```

Oh when i meant custom symbols, I meant something like something user selected that can be bought from the server's shop using serverside bucks (obtained from levelling up, server games, and NOT purchased).  So users could select to use custom symbols like ``>`` if they so choose to identify as an ~~attack helicopter~~ meme.

I'm pretty certain that there are a lot of side servers that have a shop that sell this option for users, so putting them at the very top of the userlist would be quite an inconvenience.

ATM, i've put those symbols at the right between the transition of ☆ and % (0.5 more than % and 0.5 less than ☆, so they will always be sorted amongst themselves without interfering with the other groups)